src/main/java/com/commercetools/sync/inventory/InventoryService.java

```diff
@@ -0,0 +1,59 @@
+package com.commercetools.sync.inventory;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Interface contains methods to perform sphere queries/commands related to Inventory topic.
+ * Listed operations are used in {@link InventorySync}.
+ */
+interface InventoryService {
+
+    /**
+     * Fetches all {@link InventoryEntry} objects whose sku is included in {@code skus}.
+     * Returned entries have an expanded reference to the {@code supplyChannel}.
+     *
+     * @param skus {@link Set} of sku values, used in search predicate
+     * @return {@link List} of matching entries
+     */
+    List<InventoryEntry> fetchInventoryEntriesBySkus(@Nonnull final Set<String> skus);
+
+    /**
+     * Fetches all {@link Channel} that contain role {@code "InventorySupply"}.
+     *
+     * @return {@link List} of matching channels
+     */
+    List<Channel> fetchAllSupplyChannels();
+
+    /**
+     * Creates new supply channel of role {@code "InventorySupply"} and {@code key}.
+     *
+     * @param key key of supply channel
+     * @return created {@link Channel}
+     */
+    Channel createSupplyChannel(String key);
```

Comment:
I agree this will be necessary when `InventoryService` become part of the library API. But... The purpose of `InventoryService` is to provide methods for every interaction with CTP that `InventorySync` may need. It's related to "inventory" endpoint then, but not only. Since this is designed to serve only requests specific to my sync implementation, it shouldn't be shared to end user (and it's not - it's package-private). It that case it was more convenient for me to have all methods that call CTP inside one service. You see my point? (in fact maybe I should get rid of interface entirely and leave implementation only? I left it just in case)